pkg/uki/install.go

```diff
@@ -40,17 +40,32 @@ func (i *InstallAction) Run() (err error) {
 		i.cfg.Logger.Errorf("running kairos-uki-install.pre hook script: %s", err.Error())
 	}
 
-	// Deactivate any active volume on target
-	err = e.DeactivateDevices()
-	if err != nil {
-		i.cfg.Logger.Errorf("deactivating devices: %s", err.Error())
-		return err
-	}
-	// Partition device
-	err = e.PartitionAndFormatDevice(i.spec)
-	if err != nil {
-		i.cfg.Logger.Errorf("partitioning and formating devices: %s", err.Error())
-		return err
+	if i.spec.NoFormat {
```

Comment:
I'm trying to add a test for the UKI case too here: https://github.com/kairos-io/kairos/pull/2291
to be complete.

Reply:
Manually creating the disk layout that the UKI installation creates is rather tricky:

```
[kairos@fedora ~]$ sudo blkid
/dev/sr0: BLOCK_SIZE="2048" UUID="2024-04-09-09-34-22-00" LABEL="UKI_ISO_INSTALL" TYPE="iso9660"
/dev/loop0: UUID="B9E2-794E" BLOCK_SIZE="512" TYPE="vfat"
/dev/vda2: UUID="8bfa06f9-ca4f-56dc-90c9-49cf20f4f45e" TYPE="crypto_LUKS" PARTLABEL="oem" PARTUUID="d0be3cb8-e04d-4bc9-9bde-311a516eece2"
/dev/vda3: UUID="85c39d0f-4867-5227-8334-f5eec606d9eb" TYPE="crypto_LUKS" PARTLABEL="persistent" PARTUUID="bc2d3f7d-54c6-4fc9-9918-595b19129d50"
/dev/vda1: LABEL_FATBOOT="COS_GRUB" LABEL="COS_GRUB" UUID="E9F3-549E" BLOCK_SIZE="512" TYPE="vfat" PARTLABEL="efi" PARTUUID="feb38d63-49d4-4b2d-a1d6-2f340a27e4c5"
```
We can keep the code around here but writing a test for it is rather complex (creating a layout with encrypted partitions using the tpm). I'm not sure if anyone will ever be interested in this feature in UKI mode.